modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,156 @@
+====
+
+* This update supports the `v1beta1.CustomRun` API version for custom tasks.
+* This update adds support for the `PipelineRun` reconciler to create a custom run. For example, custom `TaskRuns` created from `PipelineRuns` can now use API version `v1beta1.CustomRun` instead of `v1alpha1.Run`, if the `custom-task-version` feature flag is set to `v1beta1`, instead of the default value `v1alpha1`. 
++
+[NOTE]
+====
+You need to update the custom task controller to listen for the `*v1beta1.CustomRun` API version instead of `*v1alpha1.Run` in order to respond to `v1beta1.CustomRun` requests.
+====
+
+* This update adds a new parameter `Retries` into `v1beta1.TaskRun` and `v1.TaskRun` API versions.
+
+
+[id="triggers-new-features-1-10_{context}"]
+=== Triggers
+
+* With this update, triggers support the creation of v1 `Pipelines`, `Tasks`, `PipelineRuns`, `TaskRuns`, and v1beta1 `CustomRuns` objects.
+* With this update, GitHub Interceptor blocks a pull request trigger from being executed unless invoked by an owner or with a configurable comment by an owner. 
++
+[NOTE]
+====
+To enable or disable this update, set the value of the `githubOwners` parameter to `true` or `false` in the GitHub Interceptor configuration file.
+====
+
+* With this update, GitHub Interceptor has the ability to add a comma delimited list of all files that have changed for the push and pull request events. The list of changed files is added to the `changed_files` property of the event payload in the top-level extensions field.
+* This update changes TLS `MinVersion` to `tls.VersionTLS12` so that triggers run on {product-title} when the Federal Information Processing Standards (FIPS) mode is enabled.
+
+
+[id="cli-new-features-1-10_{context}"]
+=== CLI
+
+* This update adds support to pass a Container Storage Interface (CSI) file as workspace at the time of starting a `Task`, `ClusterTask` or `Pipeline`. 
+* This update adds v1 API support to all CLI commands associated with task, pipeline, pipeline run, and task run resources. Tekton CLI works fine with both v1beta1 and v1 APIs for these resources.
+* This update adds support for an object type parameter in the `start` and `describe` commands.
+
+
+[id="operator-new-features-1-10_{context}"]
+=== Operator
+
+* This update adds a `default-forbidden-env` parameter in optional pipeline properties. The parameter includes forbidden environment variables that should not be propagated if provided through pod templates.
+* This update adds support for custom logos in Tekton Hub UI. To add a custom logo, set the value of the `customLogo` parameter to base64 encoded URI of logo in the Tekton Hub CR. 
+* This update increments the version number of the git-clone task to 0.9.
+* With this update, you can define both the `keep` and `keep-since` parameters simultaneously for a `TaskRun` or `PipelineRun` resource in the pruner configuration file.
```

Comment:
based on the issue https://issues.redhat.com/browse/SRVKP-2998 (actual issue: https://github.com/tektoncd/cli/issues/1990)
we decided not to support `keep` and `keep-since` parameters simultaneously in 1.10 release. we may remove this line
PR: https://github.com/tektoncd/operator/pull/1446

CC @concaf @piyush-garg @rupalibehera @vdemeester @VeereshAradhya

Reply:
Removed this point from RN @jkandasa. Thanks